core/isolate.rs

```diff
@@ -1012,6 +1006,51 @@ pub mod tests {
     })
   }
 
+  #[test]
+  fn dyn_import_err2() {
```

Comment:
This test passes on master

Reply:
Yes... So the error I encountered was raised during `dyn_import_done` (SyntaxError), I'll rewrite this test case to raise the same error.